docs/beyond_basics/101-173-contributing.rst

```diff
@@ -0,0 +1,28 @@
+.. _contributing_datalad:
+
+Contributing to DataLad
+-----------------------
+
+
+DataLad is free and open source software.
+Everyone can contribute in various forms -- feature requests, questions, artwork, tutorials, code patches, bug reports, ... even follows, likes, or retweets on `Twitter <https://twitter.com/datalad>`_, or discussions in our `matrix chatroom <https://app.element.io/#/room/%23datalad:matrix.org>`_.
```

Comment:
Where we mention tutorials, we could link to the new [tutorials repository](https://github.com/datalad/tutorials) and suggest that people send a pull request to add their content there.

Reply:
a tricky part with those would be to ensure that they "still work". I guess the same applies to YouTube videos of others (as I suggested to add to our channel), but hey -- even to the ones produced by ourselves ;-)  So, it is a double edged sword.